docker/install.sh

```diff
@@ -74,11 +74,11 @@ echo "ROS 2 Build"
 if [[ ! -z "$ROS1_PACKAGES$ROS2_PACKAGES" ]]; then
     if [[ ! -z "$ROS2_PACKAGES" ]]; then
         echo "Incrementally building ROS2 packages: $ROS1_PACKAGES"
-        colcon build --install-base /opt/autoware.ai/ros/install_ros2 --build-base build_ros2 --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-above $ROS2_PACKAGES --allow-overriding $ROS2_PACKAGES
+        AUTOWARE_COMPILE_WITH_CUDA=1 colcon build --install-base /opt/autoware.ai/ros/install_ros2 --build-base build_ros2 --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-above $ROS2_PACKAGES --allow-overriding $ROS2_PACKAGES
     else
         echo "Build type is incremental but no ROS2 packages specified, skipping ROS2 build..."
     fi
 else
     echo "Building all ROS2 Autoware.AI Components"
-    colcon build --install-base /opt/autoware.ai/ros/install_ros2 --build-base build_ros2 --cmake-args -DCMAKE_BUILD_TYPE=Release
+    AUTOWARE_COMPILE_WITH_CUDA=1 colcon build --install-base /opt/autoware.ai/ros/install_ros2 --build-base build_ros2 --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Comment:
Do you know why do we require this flag when we didn't before, just for documentation?

Reply:
Sure i'll add a comment. This flag is required for the gpu based build of lidar_localizer